native/cocos/editor-support/spine-wasm/spine-wasm.cpp

```diff
@@ -4,10 +4,43 @@
 #include "util-function.h"
 #include "wasmSpineExtension.h"
 
+#include <map>
+
 using namespace spine;
 
 namespace {
     HashMap<String, SkeletonData*> skeletonDataMap{};
+
+    static void updateAttachmentVerticesTextureId(SkeletonData* skeletonData, spine::Vector<spine::String>& textureNames, spine::Vector<spine::String>& textureUUIDs) {
+        std::map<std::string, std::string> textureMap;
+        int textureSize = textureNames.size();
+        for (int i = 0; i < textureSize; ++i) {
+            textureMap[textureNames[i].buffer()] = textureUUIDs[i].buffer();
+        }
+
+        auto& skins = skeletonData->getSkins();
+        auto skinSize = skins.size();
+        for (int i = 0; i < skinSize; ++i) {
+            auto* skin = skins[i];
+            auto entries = skin->getAttachments();
```

Comment:
auto entries -> auto &entries

Reply:
Use `const auto &entries` if possible.

Reply:
compile error:  
error: non-const lvalue reference to type 'AttachmentMap::Entries' cannot bind to a temporary of type 'AttachmentMap::Entries'
   25 |             auto& entries = skin->getAttachments();

Reply:
Got it.